lighthouse-core/report/html/report-styles.css

```diff
@@ -614,6 +614,10 @@
   color: var(--color-gray-700);
 }
 
+.lh-calclink::before {
+  content: ' ';
```

Comment:
![image](https://user-images.githubusercontent.com/4071474/81757680-85323a00-9474-11ea-931a-acf10d898872.png)

can you add a space another way

Reply:
thank for catching. i tried a few ways and thought my padding solution was a bit extra. i remembered `content` but didnt test it. whoops

back to being **_extra_**.